### DISCUSSION TITLE
crit unable to find 'v1.46.2' - use 'latest'

### DISCUSSION THREAD

**Discussion by rexagod** [golangci/golangci-lint#2983] [Q&A] [answered]
Hello @ldez. I followed [your suggestion](https://github.com/golangci/golangci-lint/issues/2960), but it's still not able to find a fixed-version tag: https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/red-hat-storage_mcg-osd-deployer/61/pull-ci-red-hat-storage-mcg-osd-deployer-main-lint/1548990332694171648#1:build-log.txt%3A45-48.

(in case the above link expires: https://pastebin.com/Y55mcLXW)

Any clues?

**Comment by maratori** [comment]
@rexagod Can you enable debug logs by adding `-d` flag?
```shell
curl -JL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -d -b "$(go env GOBIN)" v1.46.2
```

**Reply by rexagod** [reply]
Getting a 429: https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/red-hat-storage_mcg-osd-deployer/71/pull-ci-red-hat-storage-mcg-osd-deployer-main-lint/1552663276620353536#1:build-log.txt%3A29. Not sure how are there too many requests being sent from our end?

**Reply by maratori** [reply, marked as answer]
You can check out [docs](https://docs.github.com/en/rest/overview/resources-in-the-rest-api#rate-limiting)
I see several options:
1. Authenticate in github
2. Use `go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.46.2`
3. Use [docker](https://hub.docker.com/r/golangci/golangci-lint)

**Reply by rexagod** [reply]
Trying these out.
